docs/content/en/docs/packages/cluster-autoscaler/addclauto.md

```diff
@@ -64,19 +50,17 @@ To read more about the tradeoffs of these different approaches, see [Autoscaling
 
```

Comment:
Make sure we capture @cxbrowne1207 's changes: https://github.com/aws/eks-anywhere/pull/6115